src/EspressoNitroTEEVerifier.sol

```diff
@@ -33,19 +37,19 @@ contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier,
         Ownable()
     {
         _certManager = certManager;
-        registeredEnclaveHash[enclaveHash] = true;
+        registeredBatchPosterEnclaveHashes[enclaveHash] = true; // TODO: modify this constructor after some review on this.
```

Comment:
does this TODO need to be addressed?

Reply:
Probably, but I think the best course of action might be to leave it as it is?
That way its possible to set up a TEEVerifier contract for a network without having to have a PCR0 hash ready for a caff node if we don't want to start a networks integration with a caff node?
Otherwise I think we can just require both?

Reply:
just curious, why does the batch poster enclave hash gets set in the constructor and not the caffnode enclave hash as well? also perhaps we can add a comment to specify that it's the batch poster enclave hash (or rename the var)